site/search/SearchPanel.tsx

```diff
@@ -397,21 +571,21 @@ const SearchResults = (props: SearchResultsProps) => {
                     </section>
                 </NoResultsBoundary>
             </Index>
-            <Index indexName={getIndexName(SearchIndexName.Explorers)}>
```

Comment:
The diff here and below is noisy, but it's really just:
- Swapping the indexes for `charts` and `explorers`
- Changing the `cutoffNumber`s
- Using the new `<ExplorerViewHits>` component

Reply:
The whole "getting 4 sets of results" thing I could finally resolve now (by cheating the Algolia types in a somewhat hacky way 😭).
I then realized that our current Algolia click tracking also didn't really work - the `queryId` is different depending on the index, but we were always sending along the id of the `pages` index.